### DISCUSSION TITLE
Can it take notes freely like the edge browser?

### DISCUSSION THREAD

**Discussion by arashrun** [sumatrapdfreader/sumatrapdf#3492] [Q&A]
Just like this below pic shown
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/45729183/d33d1a43-8bae-450c-99be-f32cb4798f4e)

**Comment by GitHubRulesOK** [comment]
Not yet possible in SumatraPDF 
So I call Edge and a save in edge shows instantly in SumatraPDF

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7240a5a1-8e29-493e-8f99-8ce95ffd3808)

```
ExternalViewers [
	[
		CommandLine = "C:\windows\system32\cmd.exe" /c Write "%1" "-Page%p.txt"
		Name = Write the Notes for &This page
		Filter = *.*
	]
	[
		CommandLine = "C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe" --app="%1#page=%p"
		Name = Edge &as Application
		Filter = *.*
	]
]
```

**Reply by arashrun** [reply]
Thanks, it can works. But it seems that %1 does not support CJK language.

**Reply by GitHubRulesOK** [reply]
yes thats generally a Windows/"Safe URL" issue hat linux or mac users may see less since MS allow windows filenames to be less traditional than older NO SPACES ETC 8.3 filename format.
 the way to allow for different code page characters would be to try passing the filename through a command file that translates the characters into URL conforming characters, but that's a much more complex task, generally since my PDF s predominantly have safe URL characters they work well with that command line.

**See if "file:///%1#page=%p" works any better**
